.github/workflows/staging-pipeline.yml

```
@@ -155,20 +148,23 @@ jobs:
-          fetch-depth: 0
+          fetch-depth: 1
       - name: Login to GitHub Container Registry
         run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+      - name: Set up Buildx
+        uses: docker/setup-buildx-action@v3
       - name: Build image and push to GitHub Packages
         uses: docker/build-push-action@v6
         with:
           push: true
-          no-cache: true
           repository: giveth/impact-graph
```

actionlint (1.7.8)

input "repository" is not defined in action "docker/build-push-action@v6". available inputs are "add-hosts", "allow", "annotations", "attests", "build-args", "build-contexts", "builder", "cache-from", "cache-to", "call", "cgroup-parent", "context", "file", "github-token", "labels", "load", "network", "no-cache", "no-cache-filters", "outputs", "platforms", "provenance", "pull", "push", "sbom", "secret-envs", "secret-files", "secrets", "shm-size", "ssh", "tags", "target", "ulimit"

(action)